source/extensions/filters/http/alternate_protocols_cache/filter.h

```diff
@@ -39,7 +39,8 @@ using FilterConfigSharedPtr = std::shared_ptr<FilterConfig>;
  * Alternate protocol cache filter which parses the alt-svc response header and updates
  * the cache accordingly.
  */
-class Filter : public Http::PassThroughEncoderFilter, Logger::Loggable<Logger::Id::forward_proxy> {
+class Filter : public Http::PassThroughEncoderFilter,
+               Logger::Loggable<Logger::Id::alternate_protocols_cache> {
```

Comment:
I'm not entirely clear if we should have a cache_filter log and a cache log, but as the only use case for this is logging something is about to go in the cache I'm inclined to leave it the same ID for now unless another maintainer objects :-)

Reply:
SGTM :)